tests/sles4sap/hana_install.pm

```diff
@@ -24,6 +24,32 @@ sub is_multipath {
     return (get_var('MULTIPATH') and (get_var('MULTIPATH_CONFIRM') !~ /\bNO\b/i));
 }
 
+=head2 download_hana_assets_from_server
+
+  download_hana_assets_from_server()
+
+Download and extract HANA installation media to /sapinst directory of the SUT.
+The media location must be provided as ASSET_0 in the job settings and be
+available as an uncompressed tar in the factory/other directory of the openQA
+server
+
+=cut
+
+sub download_hana_assets_from_server {
+    my $target = $_{target} // '/sapinst';
+    my $nettout = $_{nettout} // 2700;
+    assert_script_run "mkdir $target";
+    assert_script_run "cd $target";
+    my $filename = get_required_var('ASSET_0');
+    my $hana_location = data_url('ASSET_0');
+    # Each HANA asset is about 16GB. A ten minute timeout assumes a generous
+    # 27.3MB/s download speed. Adjust according to expected server conditions.
+    assert_script_run "wget $hana_location", timeout => $nettout;
+    assert_script_run "tar -xf $filename";
```

Comment:
We could've combined these 2 in:
```
assert_script_run "curl -k $hana_location | tar -xf -", timeout => $nettout;
```
As `tar` would fail if it doesn't get a proper tarball, and with no compression in place, the overhead added by `tar` is minimal (and we save on storage in the SUT).